tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -187,9 +187,19 @@ install_experimental_kernel() {
 
 # Install static qemu asset
 install_qemu() {
-	info "build static qemu"
 	export qemu_repo="$(yq r $versions_yaml assets.hypervisor.qemu.url)"
 	export qemu_version="$(yq r $versions_yaml assets.hypervisor.qemu.version)"
+
+	install_cached_tarball_component \
+		"QEMU" \
+		"${jenkins_url}/job/kata-containers-main-qemu-$(uname -m)/${cached_artifacts_path}" \
```

Comment:
I noticed that there are jobs for both: 
http://jenkins.katacontainers.io/view/all/job/kata-containers-main-qemu--x86_64 and
http://jenkins.katacontainers.io/view/all/job/kata-containers-main-qemu-x86_64
Is the first one needed?

Reply:
The idea once this one gets merged is to remove the latter completely, as it's only used by our "old cache mechanism".

Makes sense?

Reply:
Will the `qemu--` job be renamed then. as this is pointing to the latter at the moment?

Reply:
Yep, that's exactly the idea.  I just didn't do it right now as the PR switching to using kata-deploy local-builds is not yet merged on the tests repo.  Once that one gets merged I'll remove the original and rename the qemu-- one.